osp/core/ontology/yml/yml_parser.py

```diff
@@ -404,8 +406,51 @@ def _set_inverse(self, entity_name, entity_doc):
             (iri, rdflib.OWL.inverseOf, inverse_iri)
         )
 
-    def _check_default_rel(self, entity_name, entity_doc):
-        """Check if the given relationship the default
+    def _assert_default_relationship_occurance(self):
+        """Assures that only one default relationship is defined in the yaml
+
+        :raises ValueError: If more than one definition is found.
+        """
+        occurances = 0
+        if DEFAULT_REL_KEY in self._doc:
+            occurances += 1
+        for entity_name, entity_doc in self._ontology_doc.items():
+            if DEFAULT_REL_KEY in entity_doc:
+                occurances += 1
+        if occurances > 1:
+            raise ValueError(f"""You have defined {occurances} default
+        relationships for namespace {self._namespace}
+        although <= 1 are allowed.""")
+
+    def _check_default_rel_definition_on_ontology(self):
+        """Check if the given yaml defines
+        a default relationship, save that accordingly.
+        """
+        if DEFAULT_REL_KEY in self._doc:
+            namespace, entity_name = self._doc[DEFAULT_REL_KEY].split('.')
+
+            # defined relationship must be installed
+            from osp.core.namespaces import _namespace_registry
+            referred_namespace = _namespace_registry.get(namespace)
+            if not referred_namespace:
+                raise ValueError(f"""The namespace {namespace} that you
```

Comment:
I prefer using
```py
ValueError(f"line1 "
           f"line2 "
           f"line3 ")
```
over 
```py
ValueError(f"""line1
    line2
    line3""")
```
if the string does not fit in one line.
The first string results in `line1 line2 line3` while the latter results in

```
line1
    line2
    line3
```

Reply:
ValueError: You have defined 2 default relationships for namespace default_rel_test_namespace_two_definitions although <= 1 are allowed.

vs.

ValueError: You have defined 2 default
        relationships for namespace default_rel_test_namespace_two_definitions
        although <= 1 are allowed.

Reply:
Alright, thanks :)